### DISCUSSION TITLE
Running pre-release side-by-side

### DISCUSSION THREAD

**Discussion by simon-barratt-seq** [maester365/maester#360] [General]
Is there a recommended way to run a pre-release version of Maester?

**Comment by SamErde** [comment]
You can install a specific pre-release version by running this to target the specific preview version that you want to test:
```powershell
Install-Module -Name Maester -AllowPrerelease -RequiredVersion 0.2.26-preview
```

You can similarly import specific versions:
```powershell
Import-Module -Name Maester -RequiredVersion 0.2.26
```

**Reply by simon-barratt-seq** [reply]
Thanks. Appreciate the tip. Love the new ASCII art!

**Reply by merill** [reply]
@f-bader is our ASCII master :)
